libs/designsystem/modal/src/modal/services/alert.helper.spec.ts

```diff
@@ -86,5 +87,14 @@ describe('AlertHelper', () => {
       expect(backdrop).toHaveComputedStyle({ '--backdrop-opacity': backdropOpacity });
       await ionModalElement.dismiss();
     });
+
+    it('should set alert title text content as modal label', async () => {
+      await TestHelper.whenReady(ionModal);
+      const dialogElement = ionModal.shadowRoot.querySelector('[role="dialog"]');
+
+      await TestHelper.whenTrue(() => dialogElement.hasAttribute('aria-label'));
```

Comment:
Consider readability, waiting for the attribute is also part of "Arrange":
```suggestion
      const dialogElement = ionModal.shadowRoot.querySelector('[role="dialog"]');
      await TestHelper.whenTrue(() => dialogElement.hasAttribute('aria-label'));
```